### ISSUE TITLE
display major OS versions separately

### ISSUE THREAD

**Issue by carlwgeorge** [certsimple/rosetta-stone#21]
> All commands below are for the latest server or 'long-term' versions.

One of the big benefits of "long-term" versions is that you don't immediately have to upgrade when the next version comes out.  Because of this, enterprise environments are often one or more major versions behind, reducing the usefulness of your tool.  Both RHEL and Ubuntu currently have several different supported "long-term" versions.  Many of the referenced tasks are drastically different between these major versions.

RHEL | supported until
---|---
5 | 2017-03
6 | 2020-11
7 | 2024-06

Ubuntu | supported until
---|---
12.04 | 2017-04
14.04 | 2019-04
16.04* | 2021-04

\* not yet released

Differentiating the major versions would allow your tool to function as a transition guide between major versions of one OS.  It would also improve the primary function of cross-OS translation, since some users may only be familiar with older releases.

On a simple chart I might be concerned that this might be too much information to convey in one place, but your tool has the advantage of being able to only display only select operating systems, which is an outstanding feature.

Would this project be open to including (but differentiating) older supported versions of operating systems?

**Comment by mikemaccana** [comment]
You're totally correct re: LTS. Would be open to including older OSs - if you add them, just add them as a separate OS in the markdown file - eg, under 'Red Hat Enterprise Linux' (which currently implies RHEL 7) add 'Red Hat Enterprise Linux 6' - this way you can use 'as above' if it hasn't changed.

**Comment by carlwgeorge** [comment]
Related to this, the [logo image for RHEL](https://github.com/certsimple/rosetta-stone/blob/master/images/rosetta-stone/rhel.svg) doesn't include the version number.  Where did you get that one from?  Can you find an svg specifically for RHEL5, RHEL6, and RHEL7?  I found [this one](http://rhtbrand-cee.rhcloud.com/wp-content/uploads/2014/09/RH_logotype_variations_01.png) in the [product logo guidelines](http://brand.redhat.com/logos/productlogos/), but it's not an svg and there aren't corresponding ones for RHEL5 and RHEL6.

**Comment by carlwgeorge** [comment]
@mikemaccana ping?

**Comment by mikemaccana** [comment]
Hi @carlwgeorge, I normally get logos from either SVG or PDFs is there are no SVGs available - in this caseI can't remember where I got it. Any vector log from a PDF can be easily turned into an SVG via Sketch. If you can find one with a version number it would be rad, otherwise we'd need to get the right font and add it.
